### PR TITLE
Alex progress

### PR DESCRIPTION
**Pull request by alexling92**
1. Updated virtual cluster task with the setting up of the vm cluster in VMware.
1. Updates on the container task with set up conducted using docker.

### CODE REVIEW

**Comment by alexling92** [comment]
Btw, I accidentally use a pull request to merge the infra-in-containers folder to my branch. John explained to me that I can just update from master without needing a pull request. Sorry!

File: infra-in-containers/alex.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - Run `apt-get install docker-ce docker-compose`
+1. Verify docker-ce is installed correctly.
+    - Run `docker run hello-world`
+    - The command will download an image and runs it in a container. A message to show that the image is downloaded and run sucessfully will be printed.   
+
+### Building Apache HTTP server container on Docker
+1. Create a directory using `mkdir /docker`.
+1. Create a Dockerfile inside the /docker directory; `touch /docker/Dockerfile`.
+1. Create a sample webpage; `vim /docker/index.html`.
+1. Edit Dockerfile to include:
```
Comment (eugene-chow) [inline infra-in-containers/alex.md:30]: Good attempt to learn how to build your own Docker containers. You can also use the pre-cooked ones on Docker Hub.

File: virtual-cluster/alex.md [hunk 2]
```diff
@@ -127,3 +127,25 @@ The following shows a simple ansible script for the installation of postfix:
...
+However, a huge problem occurred after the setting up of the cluster, Proxmox-ve is unable to setup and host inner VMs. An extensive research online returns with result showing that Oracle VM Virtualbox does not support nested VM virtualization and thus proving that all the efforts are in vain...  
+### VMware Workstation
+Learning from the previous example, a research is conducted on VMware to check if it supports nested VM virtualization. The results showed that VMware supports nested virtualtization and therefore its safe to proceed with the test setup.  
+Setting up of the network connections are much more complicated as compared to Virtualbox. Once the network setup is completed, the setting up of the proxmox VMs and cluster is same as Virtualbox.  
+However, issue arises yet again during the setting up of inner VMs. When trying to start up the inner VMs, they will be stuck at the BIOS loading page with the CPU usage reaching 100%. Research online shows that this is actually a bug with hosting proxmox on VMware and the solution is to set the inner VM machine type to 2.6. Once that is done, the inner VMs will be able to boot up normally.
```
Comment (eugene-chow) [inline virtual-cluster/alex.md:146]: Wonderful! Please share your findings with John and Young Ee.

**Review by eugene-chow — APPROVED** [review]
Good work!

**Pull request merged**